guides/release/components/helper-functions.md

```diff
@@ -392,111 +441,94 @@ capitalized given name and family name as `givenName` and `familyName` instead o
 
 ### The `array` helper
 
-Using the [`{{array}}`](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/methods/array?anchor=array) helper,
+Using the [`{{array}}`](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper/methods/concat?anchor=array) helper,
 you can pass arrays directly from the template as an argument to your components.
 
-```handlebars
-<MyComponent
-  @people={{array
-    'Tom Dale'
-    'Yehuda Katz'
-    this.myOtherPerson
-  }}
-/>
+```gjs
+import { array } from '@ember/helper';
+import MyComponent from 'my-app/components/my-component';
+
+const myOtherPerson = 'George Washington';
+
+<template>
+  <MyComponent
+    @people={{array
+      'Tom Dale'
+      'Yehuda Katz'
+      myOtherPerson
+    }}
+  />
+</template>
 ```
 
 In the component's template, you can then use the `people` argument as an array:
 
-```handlebars {data-filename=app/components/my-component/template.hbs}
-<ul>
-  {{#each @people as |person|}}
-    <li>{{person}}</li>
-  {{/each}}
-</ul>
+```gjs {data-filename="app/components/my-component.gjs"}
+<template>
+  <ul>
+    {{#each @people as |person|}}
+      <li>{{person}}</li>
+    {{/each}}
+  </ul>
+</template>
 ```
 
 ### The `hash` helper
 
-Using the [`{{hash}}`](https://api.emberjs.com/ember/4.5.0/classes/Ember.Templates.helpers/methods/hash?anchor=hash)
+Using the [`{{hash}}`](https://api.emberjs.com/ember/release/classes/@ember%2Fhelper/methods/hash?anchor=hash)
 helper, you can pass objects directly from the template as an argument to your
 components.
 
-```handlebars
-<Greeting
-  @person={{hash
-    givenName='Jen'
-    familyName='Weber'
-  }}
-/>
+```gjs
+import { hash } from '@ember/helper';
```

Comment:
we should note here that this has the same behavior as the platform native `Object`

Reply:
Does it? I thought `hash` had some "interesting" behavior?

Reply:
yea, there is no interesting behavior